docs/usage/projects.md

```diff
@@ -56,21 +57,23 @@ The list of members (again a list in `.spec.members[]` using the `rbac.authoriza
 Each project member must have at least one role (currently described in `.spec.members[].role`, additional roles can be added to `.spec.members[].roles[]`). The following roles exist:
 
 * `admin`: This allows to fully manage resources inside the project (e.g., secrets, shoots, configmaps, and similar).
+* `serviceaccountmanager`: This allows to fully manage service accounts inside the project and request tokens for them.
```

Comment:
```suggestion
* `serviceaccountmanager`: This allows to fully manage service accounts inside the project namespace and request tokens for them.
```

Reply:
How? What about https://github.com/gardener/gardener/pull/5971#issuecomment-1128725966? Does it make sense to link to a document how this is done? IIRC, we should have a document somewhere which explains how to "create shoots programmatically", so probably this document should be moved here and adapted for the `TokenRequest`s?

Reply:
@rfranzke Documentation how to "create shoots programmatically" has nothing to do with this new role. I think nothing will change in the way how people call Gardener API create shoots. They will just use a different kubeconfig/token with validity
@dimityrmirchev Here we just grant permissions for the usage of the TokenRequest API. My proposal is just to provide a link to https://kubernetes.io/docs/reference/kubernetes-api/authentication-resources/token-request-v1/

Reply:
I think we should document how people can request/renew tokens. Just linking https://kubernetes.io/docs/reference/kubernetes-api/authentication-resources/token-request-v1/ is not good enough since this just describes the API but is not a real guide. Given that we are deprecating the `serviceaccounts` permissions for regular admins, we probably should explain the new way such that end-users know what needs to be done (before we eventually drop the permissions).

Reply:
@rfranzke We are not "deprecating the serviceaccounts permissions for regular admins" with this change. With this role we will enable the usage of both. The deprecation of the old will come with a different change.

Reply:
We are deprecating it (ref https://github.com/gardener/gardener/pull/5971/files#diff-7e475b851dc199193871099e011b234206f4000dbef916f8d9bd0c08636cb465R305), and it indeed makes sense to deprecate it now as we are adding the new/alternative solution. Why should we do it with a different change/later? Why are you so strongly objecting for adding the documentation to help end-users using the feature?

Reply:
I will take some time to write a guide on issuing/renewing a token. But I did not find anything related to programmatic shoot creation so maybe I will add a different document which will describe just the sa/token parts and link it to the role's description.

Reply:
@rfranzke I added an example on how a token can be generated. Should I add a deprecation notice for the `serviceaccount` permissions given to the regular "admin"? Are these labeled as breaking/user?

Reply:
Thanks! Yes, I think it makes sense to have a release note. It is not really a breaking change (yet), however we don't have another category which would raise the needed attention (unfortunately, `noteworthy` was dropped). So yeah, probably `breaking user` is the way to go then.

Reply:
Cool, thanks!